src/Aether/CustomUnits/VolatileOrganicCompoundIndex.cs

```diff
@@ -0,0 +1,59 @@
+﻿using UnitsNet;
+
+namespace Aether.CustomUnits
+{
+	public enum VolatileOrganicCompoundIndexUnit
+    {
+		IndexValue
+	}
+
+    public struct VolatileOrganicCompoundIndex : IQuantity
+    {
+		public VolatileOrganicCompoundIndex(double value, VolatileOrganicCompoundIndexUnit unit = VolatileOrganicCompoundIndexUnit.IndexValue)
+		{
+			Unit = unit;
+			Value = value;
+		}
+
+		Enum IQuantity.Unit => Unit;
+		public VolatileOrganicCompoundIndexUnit Unit { get; }
+
+		public double Value { get; }
+
+		#region IQuantity
+
+		private static readonly VolatileOrganicCompoundIndex Zero = new VolatileOrganicCompoundIndex(0, VolatileOrganicCompoundIndexUnit.IndexValue);
+
+		public QuantityType Type => QuantityType.Information;
+		public BaseDimensions Dimensions => BaseDimensions.Dimensionless;
+
+		public QuantityInfo QuantityInfo => new QuantityInfo("IndexUnit", typeof(VolatileOrganicCompoundIndexUnit),
```

Comment:
Is this supposed to return a new instance every time, or should it be a `{ get; }`?